charts/mattermost-operator/README.md

```diff
@@ -0,0 +1,73 @@
+Mattermost Operator Helm Chart
+====================================================
+
+This is the Helm chart for the Mattermost Operator. To learn more about Helm charts, [see the Helm docs](https://helm.sh/docs/). You can find more information about Mattermost Operator [here](https://github.com/mattermost/mattermost-operator/blob/master/README.md).
+
+The Mattermost Operator source code lives [here](https://github.com/mattermost/mattermost-operator).
+
+# 1. Prerequisites
+
+## 1.1 Kubernetes Cluster
+
+You need a running Kubernetes cluster v1.16+. If you do not have one, find options and installation instructions here:
+
+https://kubernetes.io/docs/home/
+
+## 1.2 Helm
+
+See: https://docs.helm.sh/using_helm/#quickstart
+
+We recommend installing Helm v3.4.0 or later.
+
+Once Helm is installed and initialized, run the following:
+
+```bash
+helm repo add mattermost https://helm.mattermost.com
+```
+
+# 2. Configuration
+
+To start, copy [mattermost-operator/charts/mattermost-operator/values.yaml](https://github.com/mattermost/mattermost-operator/blob/master/charts/mattermost-operator/values.yaml) and name it `config.yaml`. This will be your configuration file for the Mattermost Operator chart. You can used the default values that will deploy Mattermost-Operator, Mysql-Operator and Minio-Operator together (use of mysql and minio operators is not suggested for production environments) or update accordingly.
```

Comment:
```suggestion
To start, copy [mattermost-operator/charts/mattermost-operator/values.yaml](https://github.com/mattermost/mattermost-operator/blob/master/charts/mattermost-operator/values.yaml) and name it `config.yaml`. This will be your configuration file for the Mattermost Operator chart. You can use the default values that will deploy Mattermost-Operator, Mysql-Operator and Minio-Operator together (use of mysql and minio operators is not suggested for production environments) or update accordingly.
```